meinberlin/apps/plans/assets/PlansMap.jsx

```diff
@@ -68,7 +68,9 @@ class PlansMap extends Component {
   }
 
   componentDidMount () {
-    this.map = this.setupMap()
+    if (this.map === undefined) {
```

Comment:
works for me now! One thing I'm unsure about is the remaining code in `componentDidMount`, should it also be in the if statement? or instead of skipping `setupMap`, should we rather do `if(this.map !== undefined) {this.map.remove()}` in `componentWillUnmount`? I don't know what's best here

Reply:
I think having it in the unmount is cleaner and i think it has sorted out the filter behaviour but I will test and record behaviour for some filter selections on dev now and check them again after merge